lib/analytics.js

```diff
@@ -672,6 +689,7 @@ Analytics.prototype.normalize = function(msg) {
 
   // Ensure all outgoing requests include page data in their contexts.
   msg.context.page = defaults(msg.context.page || {}, pageDefaults());
+  msg.context.doNotTrack = Analytics._doNotTrack();
```

Comment:
Yep. I think the key to document for downstream integrations here is that this field is meant for "analytics" and not really about deciding whether to accept/reject this event.